notebooks/MAST/TESS/interm_tesscut_requests/interm_tesscut_requests.ipynb

```diff
@@ -0,0 +1,397 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Create a list of nearby bright stars (tess magnitude less than 14) from the rest of the data for later.\n",
+    "bright=catalogData['Tmag'] < 14\n",
+    "\n",
+    "#Make it a list of Ra, Dec pairs of the bright ones. So this is now a list of nearby bright stars.\n",
+    "nearby_stars = list( map( lambda x,y:[x,y], catalogData[bright]['ra'], catalogData[bright]['dec'] ) )\n",
+    "len(nearby_stars)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "***"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Perform a Sector Query \n",
+    "<a id=\"catalog_ID\"></a>\n",
+    "Using the TESS sector information service, we make a request to determine which sectors/cameras/CCDs contain data for this target. Remember that there is a set of FFIs for each TESS sector and those are broken up into 4 cameras which each have 4 CCDs.  We will do this with a radius=0 cone search to find only those FFI sets that contain the star of interest. You can also make the query using a larger radius, which may be important if the star is near the edge of one of the CCDs.\n",
+    "<p>\n",
+    "Note, the request is returned in a json format. The 'results' key contains an array of dictionaries with the information we are looking for.  "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "url = urlroot + \"/sector\"\n",
+    "\n",
+    "myparams = {\"ra\":Ra, \"dec\":Dec, \"radius\":\"0m\"}\n",
+    "\n",
+    "requestData = requests.get(url = url, params = myparams)\n",
+    "\n",
+    "print(requestData.headers.get('content-type'))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The resulting dictionary of information is stored in results. This target is only in sector 1."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sectors = requestData.json()['results']\n",
+    "print(sectors)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "***"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id=\"tesscut_ID\"></a>\n",
+    "## Request a FFI Cutout with Astrocut\n",
+    "Astrocut is the tool that runs the cutout service around the RA and Dec that were requested. It delivers a zipped file containing a cutout for each set of FFIs as listed above. It is also possible to request only one sector using the \"sector\" parameter.  For tesscut x refers to the CCD columns and y refers to the CCD rows. Distance can be input in a variety of units, I picked pixels (\"px\").\n",
+    "\n",
+    "<Response [200]> means that your request succeeded."
```

Comment:
Need to change <Response [200]> to < Response [200] > (spaced around Response [200]) or the whole things doesn't render.

Reply:
I don't understand why the spaces are needed?  It renders just fine in my notebook and it more closely matches what is returned by the command.

Reply:
It renders in your notebook? Huh, in mine, without the spaces it's just... gone, like all I see is "means that your request succeeded." I'll see if updating jupyter does the trick.

Reply:
went ahead and changed it to spaces.